packages/terra-date-picker/src/react-datepicker/stylesheets/datepicker.scss

```diff
@@ -0,0 +1,501 @@
+@import "variables.scss";
+@import "mixins.scss";
+
+.react-datepicker-wrapper {
+  display: inline-block;
+}
+
+.react-datepicker {
+  font-family: $datepicker__font-family;
```

Comment:
Do we need to log an issue to migrate this to using css modules?

Reply:
https://github.com/cerner/terra-framework/issues/575